#2 fizz buzz

tommygithub opened the pull request:
Updated csproj file. Give this a try.

jayhilden commented:
@tommygithub can we do a little root cause analysis on why this was missed the first time.  If you were working entirely in visual studio it would have added this to the csproj file automatically.  Were you trying to work outside VS for some reason or was there a different root cause?

tommygithub commented:
I believe it may have been when I first created the fizzbuzz.cs file. I was trying to move it from views to models, but VS wasn't letting me. So using windows explorer, I either moved it into models from there, or I deleted it from views and recreated it in models. Then reloaded VS. I wasn't getting the error you had until I exited VS and opened it through the sln file. So I deleted the csproj file, then ran the project so it would regenerate it.

jayhilden commented:
@tommygithub okay so a couple lessons here:
1. never move a file outside of visual studio, you'll always have to fix it manually then
2. instead of deleting the csproj file you can add an existing file into the csproj file by dragging the file from file explorer into the correct folder in visual studio.  This looks good now and the fizzbuzz runs.
